web/src/components/header-btn.jsx

```diff
@@ -0,0 +1,24 @@
+import React from 'react';
+
+export default class HeaderBtn extends React.Component {
+  render() {
+    if (this.props.isOpen) {
+      return (
+        <button className="header-btn header-btn-open" onClick={this.props.handleClick.bind(this)}>
```

Comment:
given that the only thing that changes between the open/closed state is the class, can we use a conditional class here? then the class name could be generate at the beginning of the render function and we can use the same template for both cases.

Reply:
Hi @MichaelKohler , I'm a little stuck here... What do you mean by conditional class? I've been reading but I don't know if we are on the same page. Should the class name be something like `{true:'header-btn-open', false:'header-btn-closed'}`?

Reply:
Similar to this, yes. The syntax should be (not tested): `className={'header-btn ' + this.props.isOpen ? 'header-btn-open' : 'header-btn-closed' }`